students/AndyKwok/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,16 @@
+#Description: Program that prints out Fizz upon division of 3 and Buzz upon division of 5
+#Author: Andy Kwok
+#Last Updated: 07/01/2018
+#ChangeLog: 
+#			v1.0 - Initialization
+
+for n in range(1, 101):
+	#prints out Fizz/Buzz/FizzBuzz accordingly by if the value can be divided equally by assigned values 
+	if n%3 is 0 and n%5 is 0:
+		print('FizzBuzz')
+	elif n%3 is 0:
+		print('Fizz')
+	elif n%5 is 0:
+		print('Buzz')
+	else:
+		print(n)
```

Comment:
couple notes:
* the code is over-indented, per pep8 you should indent by 4 spaces (no tabs); pycharm does this for you, or you can configure an editor like sublime to auto-indent correctly.
* `is` should be used with `is None`, `is True` comparison (it's an object comparison); in other cases you should use `==` equality operator
* don't forget proper spacing (having a linter would help you with that 😉 ) -> `elif n % 5 == 0`

otherwise code logic looks good!